pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -4,4 +4,18 @@ one = "kafka"
 
 [kafka.cmd.shortDescription]
 description = "Short description for command"
-one = "Create, view, use, and manage your Apache Kafka instances"
\ No newline at end of file
+one = "Create, view, use, and manage your Apache Kafka instances"
+
+[kafka.cmd.longDescription]
+description = "Long description for command"
+one = "Manage your Apache Kafka instances directly from the command line."
```

Comment:
I would write this more clearly by providing split between Kafka Management (CREATE,DELETE etc.)
and Kafka instance commands - Topics, ConsumerGroups.

```
Group of commands that lets you to manage and interact with the kafka instances.
You can manage your Apache Kafka instances by:

- kafka create, delete, list, describe
- control your currently used kafka by "kafka use"
- interact with kafka instance by `kafka topics` and `kafka consumergroup`

```


I would apply the same to the registry command in separate PR.